docs/docs.css

```diff
@@ -68,6 +63,33 @@
 /* || article || */
 .article {
   flex: 4;
-  padding: 1rem;
-  /* background-color: orange; */
+  display: flex;
+  flex-direction: column;
+  padding: 1.5rem;
+  gap: 3rem;
+}
+.article > h1 {
+  font-size: 2rem;
+}
+.inner-container {
+  display: flex;
+  flex: 1;
+  flex-direction: column;
+  gap: 1rem;
+}
+.showcase {
+  display: flex;
+  justify-content: space-around;
+  background-color: #f5f5f5;
```

Comment:
use color variable

Reply:
Thank you for pointing that out. Will correct it.